### ISSUE TITLE
Proposal: extract PHIL into a standard python package

### ISSUE THREAD

**Issue by graeme-winter** [cctbx/cctbx_project#565]
I would like to be able to use PHIL for input for a non-cctbx / non-DIALS Python project and it would be very nice to be able to 

```
pip install phil
```

or 

```
conda install phil
```

and not get the whole of cctbx etc. - currently we are using JSON for input to this and mapping to PHIL would be almost identical UI wise but much more useful. 

Is there any hostility to making PHIL a standard Python package? I can see there being takers for it as it is a lot more useful for complex input than anything else in the standard library.

**Comment by dagewa** [comment]
I like this idea. PHIL has some nice features and it would be laudable to gift that to the general Python community. The fact that "core PHIL" is extensible (e.g. to `iotbx.phil`) gives some hope that it could be extracted cleanly, but I do wonder how deep the roots go into `libtbx`. Worth a look.

**Comment by nksauter** [comment]
The experience with dxtbx was entirely unsatisfactory.  Once it was moved
out of cctbx_project, new coding practices were introduced outside of our
control here at Berkeley Lab.  In my view there are two options for PHIL:
a) keep everything exactly as it is now. b) separate PHIL into a new
repository entirely under LBNL control; still open source, but no shared
development.

**Comment by stranskj** [comment]
Yes, please. I actually was just looking on how to use Phil stand-alone... Cannot believe the issue is just 4days old :-D
Something like PHIL is terible missing in Python... There is plenty other CLI stuff, but non suitable for potentialy complex inputs. JSON is to picky to be user friendly.

Please, make it even from pip only. However, there is already 'phil' package in PyPi already.

**Comment by bkpoon** [comment]
This looks like something to be discussed at the next cctbx management meeting. A potential solution is that I can build the `conda` and `pip` packages for PHIL without changing anything in the repository.
